### PR TITLE
New Package: Soundux.Soundux version 0.2.7

### PR DESCRIPTION
**Pull request by og-mrk**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/141520)

### CODE REVIEW

**Comment by og-mrk** [comment]
This PR should satisfy issue https://github.com/microsoft/winget-pkgs/issues/58855

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 855.)

File: manifests/s/Soundux/Soundux/0.2.7/Soundux.Soundux.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /VERYSILENT, /SP-
+  Custom: /NORESTART, /LOG
+UpgradeBehavior: install
+ReleaseDate: 2021-06-04
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/s/Soundux/Soundux/0.2.7/Soundux.Soundux.installer.yaml:19]:
```suggestion
- Architecture: x64
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 856.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 856.)

**Comment by stephengillie** [comment]
Hi @og-mrk

The installation is unattended, but installs a driver which isn't unattended:
image

Unfortunately, installer switches are not usually provided for this situation. Are you aware of an installer switch to have the driver silently install as well?

**Comment by og-mrk** [comment]
Hello @stephengillie

Sadly after reading through the Original Issue requesting this package, I've noticed that the same problem is happening over here, that being The Driver Installer isn't Silently Installing when Soundux is installing, the only possible way to install programs like this one is by using Dependency Resolution, a Feature that's still not fully supported by `winget-cli` (see [this issue](https://github.com/microsoft/winget-cli/issues/163) to see why).

Anyhow, my apologies for not checking before making this PR, hope I didn't waste much of Maintainer's time, thanks for the help as always, and have a nice day.

-----

**Note**: I'm still hopeful to making this package install completely unattended, so I'll convert this PR to Draft for time been.

**Comment by og-mrk** [comment]
Like I've stated before, I'll look into this PR when  Soundux have implemented an installer that is separate from the Driver Installer.

Which unfortunately, I don't really have a clear idea as to when this will happen.

**Pull request closed without merging**